src/main/java/by/exadel/internship/mail/EmailTemplate.java

```diff
@@ -0,0 +1,16 @@
+package by.exadel.internship.mail;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class EmailTemplate {
+    private String from = "test.exadel.test@gmail.com";
```

Comment:
Looks like all fields in template are constants. If so make it static final and remove all annotations

Reply:
done